experimental/framework/fluid-static/src/FluidStaticTinylicious.ts

```diff
@@ -0,0 +1,99 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+    ITinyliciousFileConfig,
+    TinyliciousService,
+} from "@fluid-experimental/get-container";
+import { Container } from "@fluidframework/container-loader";
+import {
+    DOProviderContainerRuntimeFactory,
+    FluidContainer,
+} from "./containerCode";
+import { ContainerConfig } from "./types";
+
+/**
+ * FluidTinyliciousInstance provides the ability to have a Fluid object backed by a Tinylicious service
+ */
+export class FluidTinyliciousInstance {
+    private readonly containerService: TinyliciousService;
+
+    public constructor(port?: number) {
+        this.containerService = new TinyliciousService(port);
+    }
+
+    public async createContainer(
+        fileConfig: ITinyliciousFileConfig,
+        containerConfig: ContainerConfig,
+    ): Promise<FluidContainer> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerConfig,
+        );
+        const container = await this.containerService.createContainer(
+            fileConfig,
+            runtimeFactory,
+        );
+        return this.getRootDataObject(container);
+    }
+
+    public async getContainer(
+        fileConfig: ITinyliciousFileConfig,
+        containerConfig: ContainerConfig,
+    ): Promise<FluidContainer> {
+        const runtimeFactory = new DOProviderContainerRuntimeFactory(
+            containerConfig,
+        );
+        const container = await this.containerService.getContainer(
+            fileConfig,
+            runtimeFactory,
+        );
+        return this.getRootDataObject(container);
+    }
+
+    private async getRootDataObject(
+        container: Container,
+    ): Promise<FluidContainer> {
+        const rootDataObject = (await container.request({ url: "/" })).value;
+        return rootDataObject as FluidContainer;
+    }
+}
+
+/**
+ * Singular global instance that lets the developer define all Container interactions with the Tinylicious service
+ */
+let globalFluidTinylicious: FluidTinyliciousInstance | undefined;
+export const FluidTinylicious = {
```

Comment:
Let's give thought naming here from both the package and object perspective. We should have a convention across the Services we provide. For packages I want to propose a 1:1 mapping between the service and the import and avoid grouping multiple services in a single package. Even if they share the same driver or have the same api.

## package naming

Tinylicious will need both the client and server bits. I did a bit of research and these are the general paradigms.

I'm leaning towards #4.

1. `tinylicious` and have both client and server in the package
2. `tinylicious-client` and `tinylicious`
3. `tinylicious-client` and `tinylicious-server`
4. 1 & 3 - support both where tinylicious is a re-export of the other two for simplicity

## import naming

For the naming nothing is really jumping out. I think I like #2 but am open to more ideas

1.
```typescript
import { FluidTinylicious } from "tinylicious-client";
```

2.
```typescript
import Tinylicious from "tinylicious-client";
```

3.
```typescript
import { Fluid } from "tinylicious-client";
```

Reply:
let's rename get-container to match what we choose above.

Reply:
I'm working on creating a draft to show how FluidStaticTinylicious and TinyliciousService can be taken out of their existing locations in fluid-static and get-container respectively to be moved to a separate "tinylicious-client" package
However, we may want to stage those changes in two parts, the first where we add in the elements to the existing packages and then where we take them out into its own individual package

Reply:
I've created the draft PR with the changes to separate out `tinylicious-client` into its own package here: https://github.com/microsoft/FluidFramework/pull/5838
It doesn't remove `get-container` entirely but takes out `tinyliciousService` from it and move it into the new tinylicious-client package, which also hosts `FluidTinylicious`.
I was looking into  re-exporting out the actual Tinylicious BE server package itself from the same package as well but, as it stands, that package doesn't provide a export yet so I believe that would need to be added first followed by the re-export